source/Color.cpp

```diff
@@ -15,6 +15,13 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 #include "Color.h"
 
+#include "Preferences.h"
+
+#include <algorithm>
+#include <vector>
```

Comment:
Seems like <vector> is unused?

Reply:
Without including algorithm, CMake gets unhappy about using clamp()

Reply:
Sorry, I meant <vector>.

Reply:
I'm going to presume you meant `<vector>` and GitHub just ate the text...